controllers/profiles/getOtherProfileByUsername.js

```diff
@@ -6,7 +6,7 @@ module.exports = async (req, res) => {
     const user = await User.findOne({
       where: {username: req.params.username},
       attributes: {
-        exclude: ['human_id']
+        exclude: ['human_id', 'is_backdoor_user', 'encrypted']
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change to exclude 'is_backdoor_user' and 'encrypted' from the user attributes is a good practice for security reasons. These fields should not be exposed or used unnecessarily.
```diff
-        exclude: ['human_id']
+        exclude: ['human_id', 'is_backdoor_user', 'encrypted']
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->